shows_app/src/app/components/shared/SideBar/Logout.tsx

```diff
@@ -0,0 +1,12 @@
+import { swrKeys } from "@/fetchers/swrKeys";
+import { mutate } from "swr";
+
+export const handleLogout = () => {
+  try {
+    localStorage.removeItem("authToken");
+  } catch (error) {
+    console.error("Error removing authToken from localStorage:", error);
+  }
+
+  mutate(swrKeys.user, null);
```

Comment:
Shouldn't this be in the same try block as removing the token from storage?
If you don't remove the token from storage, mutating the user cache won't help for long as the app will manage to refetch it.